server/ansible/roles/pbench-server-activate-start-httpd/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Consolidate related roles (e.g. start-httpd and web-server should go in the same role).